docs/user/BuildWithDocker.md

```diff
@@ -70,3 +55,35 @@ cd flow
 make
 exit
 ```
+
+## Enable GUI support
+
+To use the GUI feature you will need to start the docker with the following command:
+
+```
+docker run --rm -it \
```

Comment:
It is good to have this documented.  You could add a flow/Makefile target like 'make mount' to do this more easily.